src/components/Cards/CardWithChart.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2018 - 2022 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { ComponentProps } from 'react'
+import styled from 'styled-components'
+
+import NakedChart from '../Charts/NakedChart'
+import Card from './Card'
+
+interface CardWithChartProps extends ComponentProps<typeof Card> {
+  chartSeries: ComponentProps<typeof NakedChart>['series']
+  chartColors: ComponentProps<typeof NakedChart>['colors']
```

Comment:
While not explicitly mentioned anywhere, `ComponentProps` appears ideal for higher order components where it's not easy to know what components' properties will be, but the `CardWithChard` component specifically merges two components, where this isn't a problem...

With that reasoning, I would suggest using this:

```suggestion
interface CardWithChartProps extends CardProps {
  chartSeries: NakedChart['series']
  chartColors: NakedChart['colors']
```

Reply:
So, to make sure I understood properly, you would like me to export the props from `Card` and `NakedChart` and declare types like so?

```ts
import NakedChart, { NakedChartProps } from '../Charts/NakedChart'
import Card, { CardProps } from './Card'

interface CardWithChartProps extends CardProps {
  chartSeries: NakedChartProps['series']
  chartColors: NakedChartProps['colors']
  isLoading: boolean
}
```

Reply:
Yea